README.md

```diff
@@ -37,23 +49,35 @@ Example payloads can be found in [files directory](files)
 
 ## Requirements
 
+You are welcome to use whatever amount of time you see fit on this assessment, but we do not expect
+developers to spend more than two hours implementing both parts.
+
 ### Part 1
 
-* The application should accept JSON payloads via a route
-    * When the application receives a new payload it must be validated (reference schema), parsed and (if the data is correct) stored in the database
+* The application should accept JSON payloads via a route.
+    * When the application receives a new payload it must be validated (reference schema), parsed and (if the data is correct) stored in the database.
     * The application must be able to handle validation errors. The format of the errors is your choice so long as it is consistent and adheres to HTTP protocol standards.
 
 ### Part 2
+* The application should have a retrieval endpoint to return an individual object.
 * The application must offer a way to query a list of objects based on filters.
-    * Must filter on object_id, key, and/or value
-* The application should have a retrieval endpoint to return an individual object
+    * Must filter on object_id, key, and/or value.
 
 ## Submission
 
-At the completion of your given time constaints, fully document both what you were able to complete and what you would want to do additionally if you had more time. Once completed please email your point of contact at Blue Bite a link to the repository.
+At the completion of your given time constaints, fully document both what you were able to
+complete and what you would want to do additionally if you had more time. Once completed please
```

Comment:
Maybe we can ask them to talk about how the current implementation would scale to larger numbers of reads and writes.

Reply:
Would you rather add that as a write-up, or use it as a question in the follow-up interview? Might lend itself well to digging into past experiences with scaling etc if done in person

Reply:
Fair enough we do ask this in the assessment review probably makes sense save it for that meeting.